lib/browser/api/app.js

```diff
@@ -58,6 +58,13 @@ Object.assign(app, {
   }
 })
 
+const nativeFn = app.getAppMetrics()
```

Comment:
I guess that you meant to save a reference to a function, and not to call it here.

Reply:
indeed i did!